backend/src/auth/jwt.strategy.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@nestjs/common';
+import { PassportStrategy } from '@nestjs/passport';
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { UserService } from '../user/user.service';
+import * as dotenv from 'dotenv';
+
+dotenv.config(); // Load environment variables
```

Comment:
i think its possible to import `import dotenv/config`

also its good to just put this in the main.ts file so it will load as the app starts